// simple-mind-map/src/themes/diyColor.js
import defaultTheme from 'simple-mind-map/src/themes/default';
import merge from 'deepmerge';

/**
 * @Author: 王林
 * @Date: 2021-04-11 15:22:18
 * @Desc: 经典4
 */
export default merge(defaultTheme, {
    // 连线的颜色
    lineColor: '#A9A9A9',
    // 连线的粗细
    lineWidth: 2,
    // 概要连线的粗细
    generalizationLineWidth: 2,
    // 概要连线的颜色
    generalizationLineColor: 'rgb(56, 123, 233)',
    // 背景颜色
    backgroundColor: 'rgb(241, 241, 241)',
    // 根节点样式
    root: {
        fillColor: '#F9A655',
        color: '#fff',
        fontSize: 28,
        borderRadius: 10,
        borderColor: 'transparent',
        borderWidth: 3,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    second: {
        fillColor: 'rgb(255,111,0)',
        color: '#fff',
        fontSize: 28,
        borderRadius: 10,
        borderColor: 'transparent',
        borderWidth: 3,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    // 二级节点样式
    node: {
        fillColor: 'rgb(227,44,45)',
        borderColor: 'transparent',
        borderWidth: 3,
        color: '#fff',
        fontWeight: 'bold',
        fontSize: 18,
        borderRadius: 10,
        marginX: 100,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    // 三级及以下节点样式
    fourthNode: {
        fillColor: 'rgb(206,102,102)',
        fontSize: 18,
        fontWeight: 'bold',
        color: '#fff',
        borderColor: 'transparent',
        borderWidth: 3,
        marginY: 20,
        marginX: 100,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    fifthNode: {
        fillColor: 'rgb(146,208,80)',
        fontSize: 18,
        color: '#000000',
        borderColor: 'transparent',
        borderWidth: 2,
        // marginY: 20,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    sixthNode: {
        fillColor: 'rgb(126,181,89)',
        fontSize: 18,
        color: '#000000',
        borderColor: 'transparent',
        borderWidth: 2,
        // marginY: 20,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    seventhNode: {
        fillColor: 'rgb(102,171,206)',
        fontSize: 18,
        color: '#fff',
        borderColor: 'transparent',
        borderWidth: 2,
        marginY: 50,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    eighthNode: {
        fillColor: 'rgb(132,195,198)',
        fontSize: 18,
        color: '#fff',
        borderColor: 'transparent',
        borderWidth: 2,
        marginY: 50,
        active: {
            borderColor: 'rgb(30, 53, 86)'
        }
    },
    // 概要节点样式
    generalization: {
        fillColor: 'rgb(56, 123, 233)',
        borderColor: 'rgb(56, 123, 233)',
        color: '#fff',
        borderWidth: 0,
        active: {
            borderColor: 'rgb(169, 218, 218)'
        }
    }
})
