// simple-mind-map/src/utils/constant.js
/** 
 * @Author: 王林 
 * @Date: 2021-06-24 21:42:07 
 * @Desc: 标签颜色列表 
 */
export const tagColorList = [
    {
        color: 'rgb(77, 65, 0)',
        background: 'rgb(255, 244, 179)'
    },
    {
        color: 'rgb(0, 50, 77)',
        background: 'rgb(179, 229, 255)'
    },
    {
        color: 'rgb(77, 0, 73)',
        background: 'rgb(255, 179, 251)'
    },
    {
        color: 'rgb(57, 77, 0)',
        background: 'rgb(236, 255, 179)'
    },
    {
        color: 'rgb(0, 77, 47)',
        background: 'rgb(179, 255, 226)'
    }
]

/** 
 * javascript comment 
 * @Author: 王林25 
 * @Date: 2021-07-13 15:56:28 
 * @Desc: 布局结构列表 
 */
export const layoutList = [
    {
        name: '逻辑结构图',
        value: 'logicalStructure',
        img: require('simple-mind-map/src/assets/logicalStructure.jpg')
    },
    {
        name: '思维导图',
        value: 'mindMap',
        img: require('simple-mind-map/src/assets/mindMap.jpg')
    },
    {
        name: '组织结构图',
        value: 'organizationStructure',
        img: require('simple-mind-map/src/assets/organizationStructure.jpg')
    },
    {
        name: '目录组织图',
        value: 'catalogOrganization',
        img: require('simple-mind-map/src/assets/catalogOrganization.jpg')
    }
]
export const layoutValueList = ['logicalStructure', 'mindMap', 'catalogOrganization', 'organizationStructure']

/** 
 * @Author: 王林 
 * @Date: 2021-06-24 22:58:42 
 * @Desc: 主题列表 
 */
export const themeList = [
    {
        name: '默认',
        value: 'default',
        img: require('simple-mind-map/src/assets/default.jpg')
    },
    {
        name: '脑图经典',
        value: 'classic',
        img: require('simple-mind-map/src/assets/classic.jpg')
    },
    {
        name: '小黄人',
        value: 'minions',
        img: require('simple-mind-map/src/assets/minions.jpg')
    },
    {
        name: '粉红葡萄',
        value: 'pinkGrape',
        img: require('simple-mind-map/src/assets/pinkGrape.jpg')
    },
    {
        name: '薄荷',
        value: 'mint',
        img: require('simple-mind-map/src/assets/mint.jpg')
    },
    {
        name: '金色vip',
        value: 'gold',
        img: require('simple-mind-map/src/assets/gold.jpg')
    },
    {
        name: '活力橙',
        value: 'vitalityOrange',
        img: require('simple-mind-map/src/assets/vitalityOrange.jpg')
    },
    {
        name: '绿叶',
        value: 'greenLeaf',
        img: require('simple-mind-map/src/assets/greenLeaf.jpg')
    },
    {
        name: '暗色2',
        value: 'dark2',
        img: require('simple-mind-map/src/assets/dark2.jpg')
    },
    {
        name: '天清绿',
        value: 'skyGreen',
        img: require('simple-mind-map/src/assets/skyGreen.jpg')
    },
    {
        name: '脑图经典2',
        value: 'classic2',
        img: require('simple-mind-map/src/assets/classic2.jpg')
    },
    {
        name: '脑图经典3',
        value: 'classic3',
        img: require('simple-mind-map/src/assets/classic3.jpg')
    },
    {
        name: '脑图经典4',
        value: 'classic4',
        img: require('simple-mind-map/src/assets/classic4.jpg')
    },
    {
        name: '经典绿',
        value: 'classicGreen',
        img: require('simple-mind-map/src/assets/classicGreen.jpg')
    },
    {
        name: '经典蓝',
        value: 'classicBlue',
        img: require('simple-mind-map/src/assets/classicBlue.jpg')
    },
    {
        name: '天空蓝',
        value: 'blueSky',
        img: require('simple-mind-map/src/assets/blueSky.jpg')
    },
    {
        name: '脑残粉',
        value: 'brainImpairedPink',
        img: require('simple-mind-map/src/assets/brainImpairedPink.jpg')
    },
    {
        name: '暗色',
        value: 'dark',
        img: require('simple-mind-map/src/assets/dark.jpg')
    },
    {
        name: '泥土黄',
        value: 'earthYellow',
        img: require('simple-mind-map/src/assets/earthYellow.jpg')
    },
    {
        name: '清新绿',
        value: 'freshGreen',
        img: require('simple-mind-map/src/assets/freshGreen.jpg')
    },
    {
        name: '清新红',
        value: 'freshRed',
        img: require('simple-mind-map/src/assets/freshRed.jpg')
    },
    {
        name: '浪漫紫',
        value: 'romanticPurple',
        img: require('simple-mind-map/src/assets/romanticPurple.jpg')
    }
]
